lib/registries/addon/overview/route.ts

```diff
@@ -76,23 +79,50 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-            const headTags = [...this.metaTags.getHeadTags(metaTagsData)];
+
+            this.headTags = this.metaTags.getHeadTags(metaTagsData);
+
+            // Google Structured Data
+            try {
+                const jsonLD = await this.returnStructuredData(id);
+                const scriptTag = {
+                    type: 'script',
+                    content: jsonLD,
+                    attrs: {
+                        type: 'application/ld+json',
+                    },
+                };
+                if (jsonLD) {
+                    this.headTags.push(scriptTag);
+                }
+            } catch (e) {
+                const errorMessage = this.intl.t('general.structured_data.json_ld_retrieval_error');
+                captureException(e, { errorMessage });
+            }
+
             if (provider && provider.assets && provider.assets.favicon) {
-                headTags.push({
+                this.headTags.push({
                     type: 'link',
                     attrs: {
                         rel: 'icon',
                         href: provider.assets.favicon,
                     },
                 });
             }
-            this.set('headTags', headTags);
+            this.set('headTags', this.headTags);
             this.metaTags.updateHeadTags();
         }
-
         blocker.done();
     }
 
+    async returnStructuredData(guid: string): Promise<object | undefined> {
```

Comment:
Oops I think I missed this in the previous round. `returnStructuredData` doesn't need to be an `async` function since now there is no `await` here.

Reply:
Ok, this will be removed.